docs/debug-agent.md

```diff
@@ -4,44 +4,43 @@ Clear Containers runtime launch a special virtual machine when a new pod is
 created. This VM uses a VM-agent to spawn processes on behalf of the
 pod/container(s) running inside the VM.
 
-From Clear Containers 3.0-beta release a new agent is used,
-[Clear Containers Agent](https://github.com/clearcontainers/agent):
+From Clear Containers 3.0-beta release a new agent (`cc-agent`) is used,
+[Clear Containers Agent](https://github.com/clearcontainers/agent).
 
-The Clear Containers Agent relies heavily on libcontainer used by runc 
-(standard program when running containers on bare metal).
+The Clear Containers agent relies heavily on [`libcontainer`](https://github.com/opencontainers/runc/tree/master/libcontainer) used by [`runc`](https://github.com/opencontainers/runc/) (standard program when running containers on bare metal).
```

Comment:
... used by [`runc`] (https://github.com/opencontainers/runc/), which is the standard program when running containers on bare metal.